public/app/features/alerting/unified/components/rule-viewer/Actions.tsx

```diff
@@ -1,32 +1,49 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 
 import { AppEvents } from '@grafana/data';
-import { Dropdown, LinkButton, Menu } from '@grafana/ui';
+import { ComponentSize, Menu } from '@grafana/ui';
 import appEvents from 'app/core/app_events';
 import MenuItemPauseRule from 'app/features/alerting/unified/components/MenuItemPauseRule';
+import { isAlertingRule } from 'app/features/alerting/unified/utils/rules';
 import { CombinedRule, RuleIdentifier } from 'app/types/unified-alerting';
+import { PromAlertingRuleState } from 'app/types/unified-alerting-dto';
 
 import { AlertRuleAction, useAlertRuleAbility } from '../../hooks/useAbilities';
 import { createShareLink, isLocalDevEnv, isOpenSourceEdition, makeRuleBasedSilenceLink } from '../../utils/misc';
 import * as ruleId from '../../utils/rule-id';
 import { createUrl } from '../../utils/url';
-import MoreButton from '../MoreButton';
 import { DeclareIncidentMenuItem } from '../bridges/DeclareIncidentButton';
 
-import { useAlertRule } from './RuleContext';
-
 interface Props {
+  rule: CombinedRule;
+  identifier: RuleIdentifier;
+  showCopyLinkButton?: boolean;
   handleDelete: (rule: CombinedRule) => void;
   handleDuplicateRule: (identifier: RuleIdentifier) => void;
+  onPauseChange?: () => void;
+  buttonSize?: ComponentSize;
+  hideLabels?: boolean;
 }
 
-export const useAlertRulePageActions = ({ handleDelete, handleDuplicateRule }: Props) => {
-  const { rule, identifier } = useAlertRule();
-
+/**
+ * Get a list of menu items + divider elements for rendering in an alert rule's
+ * dropdown menu
+ */
+export const useAlertRuleMenuItems = ({
+  rule,
+  identifier,
+  showCopyLinkButton,
+  handleDelete,
+  handleDuplicateRule,
+  onPauseChange,
+}: Props) => {
   // check all abilities and permissions
   const [editSupported, editAllowed] = useAlertRuleAbility(rule, AlertRuleAction.Update);
   const canEdit = editSupported && editAllowed;
 
+  const [pauseSupported, pauseAllowed] = useAlertRuleAbility(rule, AlertRuleAction.Pause);
+  const canPause = pauseSupported && pauseAllowed;
```

Comment:
We realised there was a small bug in showing "pause" option for some rules that you couldn't actually pause. This check fixes it